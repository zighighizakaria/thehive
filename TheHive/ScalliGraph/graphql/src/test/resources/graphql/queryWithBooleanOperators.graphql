query filterPerson {
    allPeople {
        peopleFilteredByName: filter(
            filter: {
                name_contains: "o"
                name_ends_with: "s"
            }
        )
        {
            peopleFilteredByAge: filter(
            filter: {
                age_gt: 30
            }
        ) {
            sort(sort: { age:incr }) {
                age {
                    toList
                }
            }
        }
        }
    }
}