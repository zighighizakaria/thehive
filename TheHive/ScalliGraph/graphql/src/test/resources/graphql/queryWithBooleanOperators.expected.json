{
  "data": {
    "allPeople": {
      "peopleFilteredByName": {
        "peopleFilteredByAge": {
          "sort": {
            "age": {
              "toList": [
                32
              ]
            }
          }
        }
      }
    }
  }
}